Library/Homebrew/cmd/pull.rb

```diff
@@ -402,7 +404,168 @@ def subject_for_bump(formula, old, new)
     subject
   end
 
+  def maybe_pbcopy(text)
+    pbcopy(text) unless ARGV.include? "--no-pbcopy"
+  end
+
   def pbcopy(text)
     Utils.popen_write("pbcopy") { |io| io.write text }
   end
+
+  # Publishes the current bottle files for a given formula to Bintray
+  def publish_bottle_files_to_bintray(f, creds)
```

Comment:
Probably just a nitpick, but I'd expect the comment and the method name to say “on Bintray” instead of “to Bintray”, as the bottles are already there and we're just changing their visibility.